recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - compilers
     - cmake
     - doxygen
-    - flake8 <3.8
+    - flake8 =3.8.4
```

Comment:
Normally I would say probably `flake8 =3.8` should be better, but I think this is probably preferable to `flake8 = 3.8` given how fickle it can be (and given how long 3.8.4 has been out). Was that the reasoning?

Reply:
Yes, the reasoning was that we have been blind-sided by unexpected upgrades to flake8 in the past.  I considered having different pins in our GitHub Actions versus here, but since this is both the developer environment and Jenkins environment, having this advanced over GHA doesn't make much sense.

Reply:
Can we make this a `>=3.8.4,<3.9` to let it float a little bit but not immediate explode when the next minor version is released?

Reply:
I don't think there will actually be another 3.8 release as far as I could tell reading through the gitlab repo for flake8, so I don't think there's too much of an issue in this instance. It does seem like they only do bug fixes on patch releases (where minor releases can have breaking behavior) - with the caveat that you could be relying on a bug and not know it, of course.